### PR TITLE
add new api documentation theme

### PR DESCRIPTION
**Pull request by mbilalsiddique1**

### CODE REVIEW

File: _sass/_layout.scss [hunk 1]
```diff
@@ -165,10 +225,78 @@
...
+
+    li.active ul { display: block; }
+
+    &.sidebar-menu-platform {
+        margin-bottom: 40px;
+
+        .sidebar-header { margin-bottom: 10px; }
+    }
+}
+
```
Comment (shaneeb) [inline _sass/_layout.scss]: @mbilalsiddique1 remove extra whitespace

File: _sass/_layout.scss [hunk 2]
```diff
@@ -234,9 +373,269 @@
...
+.btn-white:focus,
+.btn-white.focus,
+.btn-white:hover,
+.btn-white:active,
+.btn-white.active {
+    color: #585858;
+    border-color: #cacacc;
+}
+
```
Comment (shaneeb) [inline _sass/_layout.scss]: extra whitespace

File: _sass/_layout.scss [hunk 3]
```diff
@@ -234,9 +373,269 @@
...
+}
+.dropdown-menu > .active > a,
+.dropdown-menu > .active > a:hover,
+.dropdown-menu > .active > a:focus {
+    color: #fff;
+    text-decoration: none;
+    background-color: #337ab7;
+    outline: 0;
+}
+
```
Comment (shaneeb) [inline _sass/_layout.scss]: extra whitespace

**Comment by shaneeb** [comment]
Ok this looks good to me. Can't see any 'approve' button...

@kushkella Please take a look and feel free to merge if this looks good to you. Thanks.

File: pages/04-groups.markdown [hunk 4]
```diff
@@ -0,0 +1,743 @@
...
+
+```
+POST /groups
+```
+
+###### Body:
+
+```
+{
+  "name" : "group1",
```
Comment (kushkella) [inline pages/04-groups.markdown]: `,` should be removed from the end.

File: pages/04-groups.markdown [hunk 5]
```diff
@@ -0,0 +1,743 @@
...
+}
+```
+
+###### Example Request:
+```
+curl -H "X-API-KEY: 12345" -d "name=foo" "https://api.keeptruckin.com/v1/groups"
+```
+###### Sample Response:
```
Comment (kushkella) [inline pages/04-groups.markdown]: this doesn't render correctly. i think we need to add line before this.

**Comment by kushkella** [comment]
Khushnood pointed out that when scrolling the left menu moves as well to adjust. Can we stick it so that it doesn't animate?

**Pull request merged**